docs/learn/learn-guides-bounties.md

```diff
@@ -56,6 +56,19 @@ Submitting a bounty proposal will reserve
 
 ## Assign a Curator to a Bounty
 
+:::info
+
+The curator assignment call can only be executed successfully if the original bounty proposal
+funding has been executed and a new spending period has not begun. Proposers need to ensure they
+calculate a time further past the new spending period for the execution of their curator assignment
+call. Suppose your bounty has been awarded and gets funded at the end of the current spending
```

Comment:
Aren't all approved bounties funded at the end of the spending period during which they got approved?